Makefile

```diff
@@ -473,7 +475,8 @@ run: ## Runs a full node
 	$(RUNPOOLDB)
 	$(RUNEVENTDB)
 	$(RUNDACDB)
-	$(RUNL1NETWORK)
+	$(DEPLOYTOSEPOLIANETWORK)
```

Comment:
I think we should still keep the default option to use the mock L1 in the quickstart. so `make run` will use the mock L1

we can create a new command `make run-sepolia` that deploy the contract to Sepolia first.

so in the compose file, we should define two containers:

- rename `dac-setup-committee` to `dac-setup-committee-mock-l1`
- create a new one `dac-setup-committee-sepolia`